src/Security/Member.php

```diff
@@ -382,7 +382,7 @@ public function canLogIn()
      *
      * @return bool
      */
-    public function isLockedOut()
+    protected function isLockedOut()
```

Comment:
Furthermore, let's say a website has MFA or LDAP enabled.

isLockedOut could return "no"; where canLogin could return "no" as well. As the member isn't locked out, but can't login because the LDAP endpoint doesn't recognise the member, or the member doesn't have MFA enabled.
isLockedOut in that case, could still return a valid Member, even though the member can not log in.

Reducing these two endpoints to a single endpoint simplifies the flow, while still giving the developer the flexibility to override either of the methods.